### PR TITLE
PREFORMA: MediaConch integration-related changes

### PR DESCRIPTION
**Pull request by jrwdunham**

### CODE REVIEW

**Comment by sevein** [comment]
@jrwdunham, isn't a migration needed after your change?

**Comment by jrwdunham** [comment]
@sevein Yep, you're right. Fixed that.

**Comment by sevein** [comment]
👍

**Comment by sevein** [comment]
I've rebased and pushed a couple of fixes.

File: fpr/__init__.py [hunk 1]
```diff
@@ -8,4 +8,4 @@
 .. moduleauthor:: Justin Simpson <jsimpson@artefactual.com>
 """
 
-__version__ = '1.6.2'
+__version__ = '1.7.0'
```
Comment (sevein) [inline fpr/__init__.py:11]: We can do this anytime before the release but the sooner the better!

**Review by sevein — APPROVED** [review]

**Review by sevein — APPROVED** [review]
The history needs to be cleaned up but it looks great. Please keep the commit where I bump the version of the package!

**Pull request merged**